dpnp/tests/test_usm_type.py

```diff
@@ -1746,3 +1746,17 @@ def test_ix(usm_type_0, usm_type_1):
     ixgrid = dp.ix_(x0, x1)
     assert ixgrid[0].usm_type == x0.usm_type
     assert ixgrid[1].usm_type == x1.usm_type
+
+
+@pytest.mark.parametrize("usm_type_a", list_of_usm_types, ids=list_of_usm_types)
+@pytest.mark.parametrize(
+    "usm_type_cond", list_of_usm_types, ids=list_of_usm_types
+)
+def test_compress(usm_type_a, usm_type_cond):
```

Comment:
Will it be possible to move these tests as part of parametrization for `test_2in_1out` per both `test_usm_type.py` and `test_sycl_queue.py` rather than a separate test?